home-data-migration/src/main/resources/db/changelog/0.0.1/Issue33.xml

```diff
@@ -144,7 +115,7 @@
                 <constraints nullable="false"/>
             </column>
             <column name="UPDATE_DATE" type="TIMESTAMP">
-                <constraints nullable="false"/>
+                <constraints nullable="true"/>
```

Comment:
This constraint must not be nullable - true

Reply:
Done.